gspread/cell.py

```diff
@@ -40,6 +40,12 @@ def __repr__(self):
             repr(self.value),
         )
 
+    def __eq__(self, other):
+        same_row = self.row == other.row
+        same_col = self.col == other.col
+        same_value = self.value == other.value
```

Comment:
I had to set these variables to avoid linting + format issues ([W503](https://www.flake8rules.com/rules/W503.html))

Reply:
that's fine.